use-timescale/upgrades.md

```diff
@@ -12,186 +12,216 @@ import SupportMatrix from "versionContent/_partials/_migrate_self_postgres_times
+$SERVICE_SHORT outage during the upgrade. However, all connections and transactions in progress during the upgrade are 
+reset. Usually, the $SERVICE_SHORT connection is automatically restored after the reset.
 
-During a maintenance event, services with replicas perform maintenance on each
-node independently. When maintenance goes on with the primary node, the primary
-node needs to be restarted. If the restart takes more than a minute, the replica
-node is promoted to the primary, given that the replica has no replication lag.
-If the maintenance on the primary node is completed within a minute and it comes
-back online, the replica remains as a replica.
+Some minor upgrades do require some downtime. This is usually between 30 seconds and 5 minutes. If downtime is required 
+for an upgrade, $COMPANY endeavors to notify you by email ahead of the upgrade. However, in some cases, we might not be 
+able to do so. Best practice is to [schedule your maintenance window][define-maintenence-window] so that any downtime 
+disrupts your workloads as little as possible and [minimize downtime with replicas][minimize-downtime]. If there are no 
+pending upgrades available during a regular maintenance window, no changes are performed.
 
-When maintenance starts on the primary and results in the promotion of the replica
-node, the maintenance proceeds then with the newly promoted replica, following the same
-sequence. If the newly promoted replica takes more than a minute to restart, the former
-primary is promoted back. In total, the process may result in up to two minutes of write
-downtime and two failover events.
+To track the status of maintenance events, see the Timescale [status page][status-page].
 
-For more information about replicas, see the
-[replicas section][replicas-docs].
+### Minimize downtime with replicas
 
-## Non-critical maintenance updates
+Maintenance upgrades require up to two automatic failovers. Each failover takes less than a few seconds.
+$SERVICE_LONGs with [$HA_REPLICAs and $READ_REPLICAs][replicas-docs] require minimal write downtime during maintenance, 
+read-only queries keep working throughout. 
 
-Non-critical upgrades are available before the upgrade is performed
-automatically by $CLOUD_LONG. To upgrade $TIMESCALE_DB manually, run `ALTER
-EXTENSION timescaledb UPDATE` in your $SERVICE_LONG, or `Pause` and `Resume`.
-If no action is taken by the user, the upgrade is triggered in the next
-available maintenance window. You can configure the maintenance window so that
-these upgrades are started at a particular time, on a set day of the week. 
+During a maintenance event, services with replicas perform maintenance on each node independently. When maintenance is 
+complete on the primary node, it is restarted: 
+- If the restart takes more than a minute, the replica node is promoted to primary, given that the replica has no 
```

Comment:
I assume something changes here for replica sets that have more than one node - need to clarify?